content/en/docs/architecture/ci-operator.md

```diff
@@ -533,6 +533,40 @@ The _released_ `4.Y` versions of `ClusterImageSet` [manifests](https://github.co
 a tool which ensures that they points to the latest version.
 It is currently not supported that a test claims by `cluster_claim` a cluster with the version which has not been released yet.
 
+### Testing with a Cluster from HyperShift
+[HyperShift](https://hypershift-docs.netlify.app/) is now another supported method to create clusters in OpenShift CI. It is suitable for tests requires only a running cluster, for example, Optional Operator. Tests with modified OCP component are usually not suitable to use HyperShift.
```

Comment:
> Could we explain why not console and monitoring?
That's HyperShfit internal.

> Could we explain why it is different from a cluster from a pool?
This is on HyperShift documents I linked before. I can summarise a bit here.

Reply:
> That's HyperShfit internal.

A reference link would be useful if it is a learned knowledge.

> This is on HyperShift documents I linked before. I can summarise a bit here.

Something like this would be useful:
Instances only for workers are created in the cloud. The masters are running as pods in the management cluster.